plotly/dashboard_objs/dashboard_objs.py

```diff
@@ -0,0 +1,391 @@
+"""
+dashboard_objs
+==========
+
+A module which is meant to create and manipulate dashboard content.
+"""
+
+import pprint
+import copy
+#from IPython import display
+
+from plotly import exceptions, optional_imports
+from plotly.utils import node_generator
+
+IPython = optional_imports.get_module('IPython')
+
+# default variables
+master_width = 400
+master_height = 400
+container_size = master_height
+font_size = 10
+
+
+def _empty_box():
+    empty_box = {
+        'type': 'box',
+        'boxType': 'empty'
+    }
+    return empty_box
+
+
+def _box(fileId='', shareKey=None, title=''):
+    box = {
+        'type': 'box',
+        'boxType': 'plot',
+        'fileId': fileId,
+        'shareKey': shareKey,
+        'title': title
+    }
+    return box
+
+
+def _container(box_1=_empty_box(), box_2=_empty_box(), size=container_size,
+               sizeUnit='px', direction='vertical'):
```

Comment:
Anyway to fix that easily? It needs to be a `dict` so I can stick it in the dashboard.